js/ui/map.js

```diff
@@ -117,11 +145,15 @@ var Map = module.exports = function(options) {
         window.addEventListener('resize', this._onWindowResize, false);
     }
 
-    this.interaction = new Interaction(this);
```

Comment:
This property wasn't underscore-private, but it doesn't look like we've ever publicly documented it or used it in an example, so I'm fine with removing it.